editor/import/3d/resource_importer_scene.h

```diff
@@ -212,6 +213,7 @@ class ResourceImporterScene : public ResourceImporter {
 		SHAPE_TYPE_SPHERE,
 		SHAPE_TYPE_CYLINDER,
 		SHAPE_TYPE_CAPSULE,
+		SHAPE_TYPE_CONE,
```

Comment:
It is not allowed to insert values in the middle of an enum, because it will break compatibility with any content using the value after (automatic)